pyodide-build/setup.cfg

```diff
@@ -29,7 +29,7 @@ install_requires =
     build==0.7.0
     virtualenv
     pydantic>=1.10.2
-    pyodide-cli>=0.2.0
+    pyodide-cli~=0.2.1
```

Comment:
Probably let's release `pyodide-cli` 0.2.2 including #20. We can also update `auditwheel-emscripten` to 0.0.9 which renamed the entrypoint to `pyodide auditwheel`.

Reply:
Agreed. `~=0.2.1` is equivalent to `==0.2.*` so we can still merge this and it will use the latest versions once those releases are made.